src/components/PlanRoom/Board.tsx

```diff
@@ -0,0 +1,20 @@
+import { Box } from '@chakra-ui/react';
+import { DndProvider } from 'react-dnd';
+import { HTML5Backend } from 'react-dnd-html5-backend';
+import Items from './RoomTables/Item';
+
+const Board = () => (
```

Comment:
It should be named `index.tsx` (and stored in separate folder named `Board`) as @dominikmlynarczyk already pointed out in our previous conversations.

Reply:
Please apply it in all appropriate places so I don't have to write the same comment everywhere.